TLSharp.Core/TelegramClient.cs

```diff
@@ -377,8 +377,15 @@ private void OnUserAuthenticated(TLUser TLUser)
             _session.SessionExpires = int.MaxValue;
 
             _session.Save();
-        }
-
+        }
+
+        public bool IsConnected()

```

Comment:
Ok then. dealing with your policy of clean commit takes time. Maybe another time.